pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: auto_size_text_field
 description: Flutter TextField widget that automatically resizes text field to fit perfectly within its bounds.
-version: 2.2.2
+version: 2.3.2
```

Comment:
@agufagit could you please revert this version bump? I will handle it when I release the new version. Thank you!

Reply:
done